var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -393,6 +393,8 @@ def cmake_args(self):
         if '+libcxx' not in spec:
             cmake_args.append('-DLLVM_EXTERNAL_LIBCXX_BUILD:Bool=OFF')
             cmake_args.append('-DLLVM_EXTERNAL_LIBCXXABI_BUILD:Bool=OFF')
+        else:
```

Comment:
i think it would be easier to read if you do
```
if '+libcxx' in spec:
   cmake_args.append('-DCLANG_DEFAULT_CXX_STDLIB=libc++')
else:
   // old code
```